docs/user/ppl/functions/datetime.rst

```diff
@@ -711,6 +738,30 @@ Example::
     +---------------------------------+
 
 
+DAY_OF_WEEK
+-----------
+
+Description
+>>>>>>>>>>>
+
+Usage: day_of_week(date) returns the weekday index for date (1 = Sunday, 2 = Monday, ..., 7 = Saturday).
```

Comment:
`DAY_OF_MONTH` states that "The dates with value 0 such as '0000-00-00' or '2008-00-00' are invalid".
This is not included here. Does that mean that these values are valid for this and other functions below?

Reply:
I missed out on this as I based this off of the SQL documentation for these functions. I believe this isn't mentioned as having a 0 in the month/day slot is an invalid date that can't exist and doesn't work for any other datetime functions. I don't believe that that line should be added here as it would also imply that it should be added to every other description, and might be worth removing form the `DAY_OF_MONTH` function, thoughts?

Reply:
I think it is worth including this statement to all relevant functions to avoid confusion for the users. If some functions indicate this and some don't even though it applies to these functions, users may think it is safe to assume that they can have 0s as values.

Reply:
As this is a limitation of dates and would be equivalent to specifying that the month has to be 12 or less, I have decided to remove this line from all functions in the file in https://github.com/opensearch-project/sql/pull/1807/commits/cd4293df46372e8d1e36bf92807f273a04a434d6